yarn-project/end-to-end/src/fixtures/utils.ts

```diff
@@ -314,6 +314,10 @@ export type SetupOptions = {
   l1BlockTime?: number;
   /** Anvil Start time */
   l1StartTime?: number;
+  /** The anvil time where we should at the earliest be seeing L2 blocks */
+  l2StartTime?: number;
+  /** How far we should assume proven */
+  assumeProvenThrough?: number;
```

Comment:
I hate seeing this guy making its way across more parts of the codebase, but let's endure it for now